common/lib/xmodule/xmodule/modulestore/xml.py

```diff
@@ -339,9 +339,15 @@ class to use if none is specified in entry_points
         # that have a course.xml. We sort the dirs in alpha order so we always
         # read things in the same order (OS differences in load order have
         # bitten us in the past.)
+
+        if self.library:
+            self.parent_xml = 'library.xml'
```

Comment:
Can you define these (library.xml and course.xml) as constants and reference those constants other places? For example, they are currently also defined in cms/djangoapps/contentstore/views/import_export.py.